packages/design-system/src/components/Tooltip/__snapshots__/Tooltip.test.jsx.snap

```diff
@@ -7,7 +7,7 @@ exports[`Tooltip renders custom trigger component 1`] = `
   onMouseLeave={[Function]}
 >
   <a
-    aria-describedby="trigger_3"
+    aria-describedby={null}
```

Comment:
Were these ids supposed to disappear?

Reply:
Seems like our snapshot tool might need some lessons, I can revert these I think

Reply:
Actually, modifying these directly seems like a bad idea? Maybe another PR for jest typescript config work?

Reply:
Maybe get into something like : https://huafu.github.io/ts-jest/

Reply:
I was calling it out because I think it's actually a regression. I'm pretty sure these should have ids but they don't in the snapshot. The snapshot changes here are symptoms of another problem as far as I can tell

Reply:
Ah gotcha!

Reply:
I think it's null on first render because it's being set in a `useEffect` function, which runs after the first render technically https://github.com/CMSgov/design-system/pull/1451/files#diff-c9c653706166a0e772cbd7bceda432d6217948044581f3f43e489a388a3c8720R137

I tried moving the initialization value to the useRef call like
```ts
const id = useRef(props.id ?? uniqueId('trigger_'));
```
and that seemed to work in Storybook and also fix the unit test snapshots

Reply:
excellent! thanks for that fix.. updated and tested, working now